patches/minecraft/net/minecraft/item/Item.java.patch

```diff
@@ -791,13 +791,31 @@
 +     * @param armor the armor itemstack
 +     */
 +    public void onHorseArmorTick(World world, net.minecraft.entity.EntityLiving horse, ItemStack armor) {}
++    
++    @SideOnly(Side.CLIENT)
++    private net.minecraft.client.renderer.tileentity.TileEntityItemStackRenderer teisr = null;
++    
++    /**
++     * @return This Item's renderer.  Nonnull.
```

Comment:
use the annotation instead of a comment

Reply:
The package has `@MethodsReturnNonnullByDefault`, so I don't think either are necessary, actually.